src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,398 @@
+			}
+			return false
+		},
+		dragStart(e) {
+			return this.preventDefault(e)
+		},
+		dropped(e, path, disabled) {
+			/**
+			 * Don't emit if dropping is disabled.
+			 */
+			if (!disabled) {
+				this.$emit('dropped', path)
+			}
+			this.$refs.actions.opened = false
+			const crumbs = document.querySelectorAll('.crumb')
+			crumbs.forEach((f) => { f.classList.remove('over') })
+			return this.preventDefault(e)
+		},
+		dragOver(e) {
+			return this.preventDefault(e)
+		},
+		dragEnter(e, disabled) {
+			/**
+			 * Don't do anything if dropping is disabled.
+			 */
+			if (disabled) {
+				return
+			}
+			// Get the correct element, in case we hover a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.classList && target.classList.contains('crumb')) {
+					const crumbs = document.querySelectorAll('.crumb')
+					crumbs.forEach((f) => { f.classList.remove('over') })
+					target.classList.add('over')
+				}
+			}
+		},
+		dragLeave(e, disabled) {
+			/**
+			 * Don't do anything if dropping is disabled.
+			 */
+			if (disabled) {
+				return
+			}
+			// Don't do anything if we leave towards a child element.
+			if (e.target.contains(e.relatedTarget)) {
+				return
+			}
+			// Get the correct element, in case we leave directly from a child.
+			if (e.target.closest) {
+				const target = e.target.closest('.crumb')
+				if (target.contains(e.relatedTarget)) {
+					return
+				}
+				if (target.classList && target.classList.contains('crumb')) {
+					target.classList.remove('over')
+				}
+			}
+		},
+		/**
+		 * Check for each crumb if we have to hide it and
+		 * add it to the array of all crumbs.
+		 *
+		 * @param {Array} crumbs The array of all crumbs
+		 * @param {Array} newCrumbs The array of the crumbs to hide and add
+		 */
+		addCrumbs(crumbs, newCrumbs) {
+			newCrumbs.forEach((crumb, i) => {
+				if (crumb.elm && crumb.elm.classList) {
+					if (this.hiddenCrumbs.includes(crumb)) {
+						crumb.elm.classList.add('hidden')
+					} else {
+						crumb.elm.classList.remove('hidden')
+					}
+				}
+				crumbs.push(crumb)
+			})
+		},
+	},
+	render: function(createElement) {
+		// Get the breadcrumbs
+		const breadcrumbs = this.$slots.default || []
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		this.addCrumbs(crumbs, crumbs1)
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement('Actions', {
+					class: 'dropdown',
+					props: {
+						forceMenu: true,
+					},
+					ref: 'actions',
+					nativeOn: {
+						dragstart: this.dragStart,
```

Comment:
Cannot use event modifiers here?

Reply:
No, this is not possible. From the docs https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth:
```
  // Event handlers are nested under `on`, though
  // modifiers such as in `v-on:keyup.enter` are not
  // supported. You'll have to manually check the
  // keyCode in the handler instead.
  on: {
    click: this.clickHandler
  },
  // For components only. Allows you to listen to
  // native events, rather than events emitted from
  // the component using `vm.$emit`.
  nativeOn: {
    click: this.nativeClickHandler
  },
```